pkg/convert/xliff2.go

```diff
@@ -104,16 +104,27 @@ func ToXliff2(translation model.Translation) ([]byte, error) {
 		},
 	}
 
+	if translation.Original {
+		xlf.SrcLang = translation.Language
+	} else {
+		xlf.TrgLang = translation.Language
```

Comment:
Does xliff12 have only TrgLang or both when it is translated?

* when original, only SrcLang is set
* when not original, SrcLang and TgtLang are set

Reply:
This is just a minor fix, addressing the failing integration tests. The implementation is incomplete for convert to XLIFF. In the current state it's not accurate/incomplete. I have added a ticket:
https://github.com/orgs/translate-agent/projects/1?pane=issue&itemId=45498477